components/scatterplot.py

```diff
@@ -0,0 +1,35 @@
+import dash_html_components as html
+import dash_core_components as dcc
+
+from utils import earthquake_data
+
+
+def get_graph(session_id):
+    """Returns an empty scatterplot graph, if data has been uploaded.
+    Otherwise returns a text informing the user, that no data was found.
```

Comment:
According to PEP8 function docstrings should be written in imperative ("Return an ...")